src/styles/styles.css

```diff
@@ -502,34 +505,39 @@ section.examples-lists > div {
 
 a.button-link {
   flex-grow: 1;
-  font-size:15px;
+  font-size: 16px;
   color: var(--button-link-color);
   background-color: var(--button-link-background-color);
-  border: 1px solid #302F45;
+  border: none;
   border-radius: 10px;
   font-family: 'Open Sans', sans-serif;
   text-decoration: none !important;
   margin: 10px 5px;
   padding: 24px 24px;
   display: inline-flex;
+  font-weight: 600;
```

Comment:
So I think that this definition takes away the differences between the title/text lines in button-link.descriptive. The way it was previously was the title (first line) was bold, and the following line was plain text. This is probably a consequence some poorly written inheritance on my part.